recipe/build.sh

```diff
@@ -4,13 +4,14 @@ mkdir build
 cd build
 
 cmake \
-  -DCMAKE_INSTALL_PREFIX=$PREFIX \
-  -DCMAKE_PREFIX_PATH=$PREFIX \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DCMAKE_PREFIX_PATH=${PREFIX} \
   -DCMAKE_INSTALL_RPATH=${PREFIX}/lib \
   -DINSTALL_RPATH_USE_LINK_PATH=ON \
   -DCMAKE_EXE_LINKER_FLAGS="-Wl,-rpath,${PREFIX}/lib -L${PREFIX}/lib" \
   -DCMAKE_SHARED_LINKER_FLAGS="-Wl,-rpath,${PREFIX}/lib -L${PREFIX}/lib" \
   -DCMAKE_BUILD_TYPE=Release \
+  -DCLING_INCLUDE_PATHS=${PREFIX}/${HOST}/include/c++/7.3.0:${PREFIX}/${HOST}/sysroot/usr/include \
```

Comment:
I've checked and `-dumpversion` works with both gcc and clang:

```suggestion
  -DCLING_INCLUDE_PATHS=${PREFIX}/${HOST}/include/c++/$(${CXX} -dumpversion):${PREFIX}/${HOST}/sysroot/usr/include \
```

Reply:
Thanks! Just added a commit.

Reply:
Ah, I did not see I could integrate your suggested change instead. Just did that.